src/main/java/com/minecolonies/coremod/network/messages/BuildToolPlaceMessage.java

```diff
@@ -142,18 +145,21 @@ private static void handleHut(
             return;
         }
 
+        final Block block = Block.getBlockFromName(Constants.MOD_ID + ":blockHut" + hut);
         final Colony tempColony = ColonyManager.getClosestColony(world, buildPos);
-        if (tempColony != null && !tempColony.getPermissions().hasPermission(player, Permissions.Action.MANAGE_HUTS))
+        if (tempColony != null
```

Comment:
It would look better I also thought about that. But we would lose the context. It would be harder to interprete.